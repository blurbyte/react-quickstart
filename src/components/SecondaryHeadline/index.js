import styled from 'styled-components';

const SecondaryHeadline = styled.h2`
  font-family: Arial, Helvetica, sans-serif;
  font-size: 1.4rem;
  color: #444;
  line-height: 1.2;
  margin: 1.4rem 0;
`;

export default SecondaryHeadline;
