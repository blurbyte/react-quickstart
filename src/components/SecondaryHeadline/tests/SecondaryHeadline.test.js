import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import SecondaryHeadline from '../index';

describe('<SecondaryHeadline />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <SecondaryHeadline />
    );

    expect(component.toJSON()).toMatchStyledComponentsSnapshot();
  });
});
