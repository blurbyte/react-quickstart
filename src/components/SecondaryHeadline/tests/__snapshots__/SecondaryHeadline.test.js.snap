// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SecondaryHeadline /> should render correctly 1`] = `
.fgsRWY {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 1.4rem;
  color: #444;
  line-height: 1.2;
  margin: 1.4rem 0;
}

<h2
  className="fgsRWY"
/>
`;
